pkg/resources/manifests/deploy.yaml

```diff
@@ -102,6 +102,10 @@ spec:
         # (For reference, see: https://github.com/inspektor-gadget/inspektor-gadget/runs/3966318270?check_suite_focus=true#step:20:221)
         container.apparmor.security.beta.kubernetes.io/gadget: "unconfined"
         inspektor-gadget.kinvolk.io/option-hook-mode: "auto"
+        # keep aligned with values in pkg/operators/prometheus/prometheus.go
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "2223"
```

Comment:
Does this work without listing the [port](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports) in our gadget pod ?

Reply:
Yes, it does! I think listing the ports is only needed when endpoints or service is configured.

Reply:
I added some new documentation that uses this approach. Could you check this out?